oscaralanpierce/skyrim_inventory_management_frontend
# Add and configure Storybook

## Context

[**Create stock Vite app**](https://trello.com/c/UF2WLo0M/237-create-stock-vite-app)

As we're building out the new front end, we need to make sure all our tooling is in place. One of the tools we need is Storybook.

## Changes

* Add Storybook packages
* Configure Storybook
* Add commands to build Storybook and run the server

## Considerations

I got a very cryptic error when attempting to run Storybook initially. However, the error turned out to be easily googleable and the solution was to prefix the Storybook commands with `NODE_OPTIONS=--openssl-legacy-provider`. Doing that completely solved it.

## Manual Test Cases

Clone the repository locally and install dependencies with `yarn`. Next, run `yarn storybook`. The Storybook server should run on port 6006. Note that it will show you an error message indicating you have no stories.
